sql/analyzer/resolve_columns.go

```diff
@@ -444,7 +444,7 @@ func getAvailableNamesByScope(n sql.Node, scope *plan.Scope) availableNames {
 			case *plan.TableAlias:
 				switch t := n.Child.(type) {
 				case *plan.ResolvedTable, *plan.UnresolvedTable, *plan.SubqueryAlias,
-					*plan.RecursiveTable, *plan.IndexedTableAccess:
+					*plan.RecursiveTable, *plan.IndexedTableAccess, sql.TableFunction:
```

Comment:
i know we just made all table functions aliases, but we should still check for `sql.TableFunction` on its own (line 440)